src/javascript/app/pages/user/account/settings/account_closure.js

```diff
@@ -371,9 +372,25 @@ const AccountClosure = (() => {
                 $submit_loading.setVisibility(0);
                 $closure_container.setVisibility(0);
                 $success_msg.setVisibility(1);
+                $('.client_logged_in, #client-logged-in, #topbar-msg').setVisibility(0);
+                $('#menu-top').removeClass('smaller-font top-nav-menu');
+                $('.client_logged_out').setVisibility(1);
+                $('topbar').addClass('primary-bg-color-dark');
+                $('topbar').removeClass('secondary-bg-color');
                 $.scrollTo(0, 500);
 
                 sessionStorage.setItem('closingAccount', 1);
+
+                // If the user clicks on a link while not having been logged out yet, log the user out.
+                $('a').on('click', () => {
```

Comment:
Selecting  tag `a` is global, are you sure we have to log the user out on select any link on the page?

Reply:
In this case I was thinking yes, since the user is still logged in technically until he/she clicks something or gets logged out automatically after 10 seconds. This selector is only relevant when the user is already on the deactivated account page.

Reply:
(optional) I think if we need to show a `message` or any other information about that the account is closed, we can handle it using a promise instead of timeout , or we can logout user and redirect them to a page to show them the message (product team agreement is required). In all cases we don’t need to add a event to `a` tag globally.

Reply:
*by promise I mean a modal with a CTA, closing modal is equal to CTA

Reply:
I created a modal after the user deactivates his account, where the user clicks on a button to close it. After that the user gets redirected automatically to the home page.